app/Views/errors/cli/error_404.php

```diff
@@ -1,6 +1,11 @@
 <?php
 
 use CodeIgniter\CLI\CLI;
+use CodeIgniter\Debug\Exceptions;
+
+if (Exceptions::isUsingPhpUnitErrorHandlingInSeparateProcess()) {
+    return;
+}
```

Comment:
I don't still understand what's happening.
Can you explain what is the exact cause of the errors?
This hack is very bad, because the view file should not know about PHPUnit process isolation.
Can't we avoid to do like this?